compiler/rustc_codegen_llvm/src/coverageinfo/ffi.rs

```diff
@@ -164,6 +164,15 @@ impl CounterMappingRegion {
                 end_line,
                 end_col,
             ),
+            MappingKind::Branch { true_term, false_term } => Self::branch_region(
```

Comment:
Can we remove the `#[allow(dead_code)]` decorator for the `branch_region()` definition then ?